src/commands/click.coffee

```
@@ -3,4 +3,7 @@ nodeify = require 'nodeify'
 
 
 module.exports = new Nifty.Command 'click', (selector, done) ->
-  nodeify @findElement(selector).click(), done
+  try
```

I might be missing something here. Is this only to catch the error that we throw in `createSelector`? If so, we should convert this method to have an async API that returns the error in the "Node way". Usually throwing exceptions doesn't mix well with the Node paradigm, unless we want to roll over and die right there as a result. If we want to catch, convert, or reformat the error, its better to stay with async APIs, and let the error bubble up the call chain.